lib/init.g

```diff
@@ -466,10 +466,17 @@ BindGlobal( "ShowKernelInformation", function()
   else
     gap := "GAP";
   fi;
+  # Turn off print formatting temporarily to avoid line wrapping caused by our
+  # possible use of unicode characters in btop and bbot which confuses the
+  # code in io.c tracking the current cursor position: those Unicode
+  # characters consist of multiple bytes each, which that code does not take
+  # into account.
+  SET_PRINT_FORMATTING_STDOUT(false);
   Print( " ",btop,"   ",gap," ", GAPInfo.BuildVersion,
          sysdate, "\n",
          " ",vert,"  GAP  ",vert,"   https://www.gap-system.org\n",
          " ",bbot,"   Architecture: ", GAPInfo.Architecture, "\n" );
+  SET_PRINT_FORMATTING_STDOUT(true);
```

Comment:
Of course it would be better to save and restore the current state -- but that requires PR #4526.